samples/unit-tests/series-heatmap/members/demo.js

```diff
@@ -210,12 +211,34 @@ QUnit.test('seriesTypes.heatmap.pointClass.setState', function (assert) {
         0,
         'When state:normal zIndex is 0'
     );
+
     setState.call(point, 'hover');
     assert.strictEqual(point.graphic.zIndex, 1, 'When state:hover zIndex is 1');
+
     setState.call(point, 'select');
     assert.strictEqual(
         point.graphic.zIndex,
         0,
         'When state:select zIndex is 0'
     );
+
+    setState.call(point, '');
+    assert.strictEqual(
+        point.graphic.d.split(' ')[1] - point.graphic.getBBox().x,
+        point.series.options.borderRadius,
+        'Border radius should have the value set in options on state:normal'
+    );
+
+    setState.call(point, 'hover');
+    assert.strictEqual(
+        point.graphic.d.split(' ')[1] - point.graphic.getBBox().x,
+        point.series.options.borderRadius,
+        'Border radius should have the value set in options on state:hover'
+    );
+
```

Comment:
I think `setState.call(point, '');` is missing here

Reply:
that's true, thanks!